README.md

```diff
@@ -42,22 +42,19 @@ classifier information.
 
 ## Messages
 
-  * Classification2D and Classification3D: pure classification without pose
+  * Classification: pure classification without pose
   * Detection2D and Detection3D: classification + pose
-  * XArray messages, where X is one of the four message types listed above. A
+  * XArray messages, where X is one of the two message types listed above. A
```

Comment:
Should this be "three" message types (Classification, Detection2D, Detection3D)?

Reply:
There's no ClassificationArray (and I think that's right, what would that be for?)

Reply:
Oh right. I just saw that you removed one message, so I assumed it should go from "four" to "three", but you're absolutely right - it should already have said "two" before.